### DISCUSSION TITLE
'Passed Output Does Not Belong To The Wallet' error message

### DISCUSSION THREAD

**Discussion by plebfreedom** [Ride-The-Lightning/RTL#1377] [Troubleshooting Questions]
Pending Channel with fee set too low is stuck in mempool and unable to bump fee inside the menu option.  It gives an error message from the same wallet I funded to open a peer channel:  

Transaction:  fff1fdb3de17c0fca822a0d98bdcd6a18d7762aa01bcd2b378e9384498e750c4

**To Reproduce**

1. Go to 'Lightning' menu, 'Peers' tab under 'Connections', select 'Actions' drop down menu and then 'Open Channel' for a successful, completed peer transaction.
2. I set drop down to 800,000 sats channel funding, clicked on 'Advanced Options', set fee to 1 sat.  
3. Look at the transaction in mempool, unconfirmed.  
4. Attempt to bump fee inside 'Pending Open Channel'.  
5. Index for Change Output is not there, but '0' seems to be the most likely.
6. Then, error: 500 - {“Code”:2,“message”:“the Passed Output Does Not Belong To The Wallet”,“details”:}

**Screenshots**
![image](https://github.com/Ride-The-Lightning/RTL/assets/163596175/1bcd7945-6450-446a-b899-5e0ee430c87a)

**Your environment**
* Version of `RTL` 0.15.0
* Version of `lnd` is 0.17.4
* Version of `electrs' 0.10.3
* Browser is Brave Version 1.61.116 Chromium: 120.0.6099.217 
* Operating system start9 with Mac Sonoma 14.2.1 local laptop

**Additional context**
Channel was opened last night.  During the night, power was disrupted.  Everything rebooted just fine, but when I logged back into RTL, it required a different password which I pulled out of system information.

**Comment by saubyk** [comment]
Hi @plebfreedom your channel opening transaction doesn't have a change output. You'll not be able to fee bump without a change output. Also keep in mind that when you're fee bumping this way, you need to provide the index of the change output not the original transaction.

https://mempool.space/tx/fff1fdb3de17c0fca822a0d98bdcd6a18d7762aa01bcd2b378e9384498e750c4

<img width="1124" alt="image" src="https://github.com/Ride-The-Lightning/RTL/assets/39208279/e5a6d45a-ab1b-440b-8474-60cc774fe7b2">

**Comment by plebfreedom** [comment]
Thank you, @saubyk.  Shall I write this off as a donation to the Bitcoin ecosystem?  Or will it miraculously get confirmed someday?

**Comment by saubyk** [comment]
You have two options now...
1. Be patient and wait in hope that mempool rates go down and the transaction gets picked up
2. Try an external transaction acceleration like the mempool.space accelerator or https://www.viabtc.com/tools/txaccelerator

For future reference, please keep in mind that external acceleration is not a free option always available and can be expensive as well. So it's always better to check the mempool and use the appropriate fee rate when opening channels

**Comment by saubyk** [comment]
Moving this issue to a discussion thread. As no development action is expected here.
